### DISCUSSION TITLE
调研bfe能否将上层服务的响应参数传递给下层服务

### DISCUSSION THREAD

**Discussion by zhhOceanfly** [bfenetworks/bfe#1027] [Q&A (问题和回答)]
还在调研bfe能否实现某一个需求，比如客户端请求统一到某个网关服务处理好请求参数，然后将服务处理得到的响应参数再传递给下层服务上，想问问看bfe能否实现这个呢？

**Comment by iyangsj** [comment]
@zhhOceanfly 没有看懂描述的需求，需求场景能否进一步描述得更清楚？
